OCC.Service.Webhost/Repositories/SessionRepository.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using OCC.Data;
+using OCC.Service.Webhost.Services;
+using OCC.Service.Webhost.Tools;
+using Session = OCC.Service.Webhost.Services.Session;
+
+namespace OCC.Service.Webhost.Repositories
+{
+    public class SessionRepository
+    {
+        private readonly OCCDB _dbContext;
+
+        public SessionRepository(OCCDB dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public Session GetSession(int id)
+        {
+            var s = (from x in _dbContext.Sessions.Include("Speaker").Include("Track").Include("Timeslot")
+                        where x.ID == id
+                        select x).FirstOrDefault();
+
+            if (s == null) throw new ArgumentException("Session not found");
```

Comment:
I actually started to change this and backed it out to start putting tests around it. This matches what is currently in the svc file on master.

I am always in favor of still using curly braces even when optional